packages/istanbul-lib-source-maps/lib/map-store.js

```diff
@@ -190,15 +190,17 @@ class MapStore {
                         return null;
                     }
 
-                    const smc = new SourceMapConsumer(obj);
+                    const smc = new TraceMap(obj);
                     smc.sources.forEach(s => {
-                        const content = smc.sourceContentFor(s);
-                        if (content) {
-                            const sourceFilePath = pathutils.relativeTo(
-                                s,
-                                filePath
-                            );
-                            this.sourceStore.set(sourceFilePath, content);
+                        if (s) {
```

Comment:
[types](https://github.com/jridgewell/trace-mapping/blob/5ccfcfeeee9dfa3b13567bb0f95260ea32f2c269/src/types.ts#L8) says `s` is `string | null`

Reply:
Correct. If sources content is enabled, some tools will use `null` if the file's source content isn't available. So some files in the array will have source content (a string), and some won't (null).